Formula/zsh-completions.rb

```diff
@@ -35,7 +35,7 @@ def caveats
       Additionally, if you receive "zsh compinit: insecure directories" warnings when attempting
       to load these completions, you may need to run this:
 
-        chmod -R go-w '#{HOMEBREW_PREFIX}/share/zsh'
+        chmod -R go-w '#{HOMEBREW_PREFIX}/share'
```

Comment:
I do not think this is acceptable. `#{HOMEBREW_PREFIX}/share` is a homebrew top directory, we should not recommend messing with its permissions.

Reply:
Thanks for the note! I'll close the PR, but do want to note that a user may continue to receive the

> zsh compinit: insecure directories

warning, and running compaudit shows `#{HOMEBREW_PREFIX}/share` which is why I suggested this.

I would love it if there was some better way to help others who run into this, as the caveat is helpful.